packages/vaadin-grid/src/interfaces.d.ts

```diff
@@ -43,37 +43,37 @@ export interface GridFilter {
   value: string;
 }
 
-export interface GridEventContext {
+export interface GridEventContext<TItem> {
   section: 'body' | 'header' | 'footer' | 'details';
-  item?: GridItem;
-  column?: GridColumnElement;
+  item?: TItem;
+  column?: GridColumnElement<TItem>;
   index?: number;
   selected?: boolean;
   detailsOpened?: boolean;
   expanded?: boolean;
   level?: number;
 }
 
-export type GridHeaderFooterRenderer = (
+export type GridHeaderFooterRenderer<TItem> = (
   root: HTMLElement,
-  column?: GridColumnElement
+  column?: GridColumnElement<TItem>
 ) => void;
 
-export type GridItem = unknown;
+export type DefaultGridItem = any;
```

Comment:
We generally name things so that all the `vaadin-grid` stuff starts with `Grid`.

Do you think it would make sense to use `GridDefautltem` instead?

Reply:
Sure, I renamed it.